src/devices/rubicson_48659.c

```diff
@@ -161,10 +162,18 @@ static int rubicson_48659_decode(r_device *decoder, bitbuffer_t *bitbuffer)
     // more then 25 repeats are not uncommon
     row = bitbuffer_find_repeated_row(bitbuffer, 10, 32);
     if (row < 0)
-        return 0;
+        return DECODE_ABORT_EARLY;
 
     if ((bitbuffer->bits_per_row[row] > 33) || (bitbuffer->bits_per_row[row] < 10))
-        return 0;
+        return DECODE_ABORT_LENGTH;
+
+    // Verify checksum
+    uint8_t sum = (add_bytes(bb[row], 3) & 0xff) - bb[row][0];
```

Comment:
The comment talks about XOR, should that be corrected?

Reply:
Well I wrote the code and I don't remember anything of it. Then this weekend I ran revdgst and this line popped out. So I don't really know, I will test with the live sensor an adapt the comment accordingly.